docs/concepts/jcloud/yaml-spec.md

```diff
@@ -34,32 +32,46 @@ Since each Executor has its own business logic, it may require different cloud r
 In JCloud, you can pass highly customizable, finely-grained resource requests for each Executor using the `jcloud.resources` argument in your Flow YAML.
 
 
-### CPU
+### Instance
 
-By default, `0.1 (1/10 of a core)` CPU is allocated to each Executor. You can use the `cpu` argument under `resources` to change it.
+JCloud uses the concept of "Instance" to respresent the hardware specifications. 
```

Comment:
```suggestion
JCloud uses the concept of an "Instance" to represent a specific set of hardware specifications. 
```
Is this correct?

Also, please spell check next time :sweat_smile:

Reply:
Yea that's correct, oops sorry I should have installed the spelling checker in the IDE.

Reply:
fixed